### PR TITLE
Repository cleanup

### PR DESCRIPTION
**Pull request by isuftin**

### CODE REVIEW

File: nhmusgs-base/Dockerfile [hunk 1]
```diff
@@ -1,34 +1,32 @@
...
-FROM continuumio/miniconda3:latest
+FROM continuumio/miniconda3:4.7.10
 
 LABEL maintainer="ashalper@usgs.gov"
 
-# ...check for any Debian, installed packages updates, upgrade them and install
-# additional packages
-RUN apt-get update && apt-get -y upgrade && \
+# Update debian repository cache, install additional packages
+RUN apt-get update && \
```
Comment (ashalper-usgs) [inline nhmusgs-base/Dockerfile]: Are you sure you want to omit the `apt-get -y upgrade` part here? Sometimes this installs security patches.

Comment (isuftin) [reply nhmusgs-base/Dockerfile]: This also may introduce unintended libraries you're not expecting to be upgraded. So it's a matter of weighing the possibilities. Considering this is a container that runs an ephemeral process in a closed environment with no outside access to it, I'm willing to let that go to maintain stability across builds. Thoughts?

Comment (ashalper-usgs) [reply nhmusgs-base/Dockerfile]: Yeah, that's fine.

**Comment by isuftin** [comment]
re-testing locally. will create new PR when complete

**Comment by ashalper-usgs** [comment]
@isuftin , are you seeing this error?
```
Running nhm-prms...
--env: /usr/local/bin/prms: Permission denied
```

**Comment by isuftin** [comment]
@ashalper-usgs Yep I am so I did close this PR without merging so I can fix these issues locally

**Pull request closed without merging**